### PR TITLE
added text and tooltip to idp

### PR DESCRIPTION
**Pull request by edewit**
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

**Review by agagancarczyk — DISMISSED** [review]

File: themes/src/main/resources/theme/keycloak.v2/login/login.ftl [hunk 1]
```diff
@@ -119,63 +124,75 @@
                                     <#switch p.alias>
                                         <#case "google">
                                             <svg aria-hidden="true" viewBox="0 0 488 512">
+                                                <title id="title">Google</title>
                                                 <path d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"/>
                                             </svg>
                                             <#break>
                                         <#case "github">
                                             <svg aria-hidden="true" viewBox="0 0 496 512">
+                                                <title id="title">Github</title>
```
Comment (ahus1) [inline themes/src/main/resources/theme/keycloak.v2/login/login.ftl:133]:
```suggestion
                                                <title id="title">GitHub</title>
```

File: themes/src/main/resources/theme/keycloak.v2/login/login.ftl [hunk 2]
```diff
@@ -119,63 +124,75 @@
...
                                             <#break>
                                         <#case "facebook">
                                             <svg aria-hidden="true" viewBox="0 0 448 512">
+                                                <title id="title">Facebook</title>
                                                 <path d="M448 56.7v398.5c0 13.7-11.1 24.7-24.7 24.7H309.1V306.5h58.2l8.7-67.6h-67v-43.2c0-19.6 5.4-32.9 33.5-32.9h35.8v-60.5c-6.2-.8-27.4-2.7-52.2-2.7-51.6 0-87 31.5-87 89.4v49.9h-58.4v67.6h58.4V480H24.7C11.1 480 0 468.9 0 455.3V56.7C0 43.1 11.1 32 24.7 32h398.5c13.7 0 24.8 11.1 24.8 24.7z"/>
                                             </svg>
                                             <#break>
                                         <#case "gitlab">
                                             <svg aria-hidden="true" viewBox="0 0 512 512">
+                                                <title id="title">Gitlab</title>
```
Comment (ahus1) [inline themes/src/main/resources/theme/keycloak.v2/login/login.ftl:145]:
```suggestion
                                                <title id="title">GitLab</title>
```

File: themes/src/main/resources/theme/keycloak.v2/login/login.ftl [hunk 3]
```diff
@@ -119,63 +124,75 @@
...
                                         <#case "linkedin-openid-connect">
                                             <svg aria-hidden="true" viewBox="0 0 448 512">
+                                                <title id="title">LinkedIn</title>
                                                 <path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/>
                                             </svg>
                                             <#break>
                                         <#case "openshift-v3">
                                         <#case "openshift-v4">
                                             <svg viewBox="116 0 100 100" aria-hidden="true">
+                                                <title id="title">Openshift</title>
```
Comment (ahus1) [inline themes/src/main/resources/theme/keycloak.v2/login/login.ftl:159]:
```suggestion
                                                <title id="title">OpenShift</title>
```

File: themes/src/main/resources/theme/keycloak.v2/login/login.ftl [hunk 4]
```diff
@@ -119,63 +124,75 @@
...
                                                 <path d="M145.7,45.3l-16.1,5.8c0.2,2.6,0.6,5.1,1.3,7.6l15.3-5.6C145.7,50.6,145.5,47.9,145.7,45.3M216.7,27.5c-1.1-2.3-2.4-4.5-3.9-6.6l-16.1,5.8c1.9,1.9,3.4,4.1,4.7,6.4L216.7,27.5zM181.4,23c3.3,1.6,6.2,3.7,8.7,6.2l16.1-5.8c-4.4-6.2-10.5-11.5-17.9-14.9c-22.9-10.7-50.3-0.7-61,22.2c-3.5,7.4-4.8,15.3-4.1,23l16.1-5.8c0.3-3.5,1.1-7,2.7-10.3C148.7,22.5,166.4,16,181.4,23M131.9,58.4l-15.3,5.6c1.4,5.6,3.8,10.8,7.2,15.5l16-5.8C135.8,69.4,133,64.1,131.9,58.4M198.5,52.3c-0.3,3.5-1.1,7-2.7,10.3C188.8,77.5,171,84,156.1,77c-3.3-1.6-6.3-3.7-8.7-6.2l-16,5.8c4.4,6.2,10.5,11.5,17.9,14.9c22.9,10.7,50.3,0.7,61-22.2c3.5-7.4,4.7-15.3,4.1-22.9L198.5,52.3zM202.4,32.7l-15.3,5.6c2.8,5.1,4.2,10.9,3.7,16.8l16-5.8C206.5,43.5,204.9,37.9,202.4,32.7"></path>
                                             </svg>
                                             <#break>
                                         <#case "stackoverflow">
                                             <svg viewBox="0 0 384 512" aria-hidden="true">
+                                                <title id="title">Stackoverflow</title>
```
Comment (ahus1) [inline themes/src/main/resources/theme/keycloak.v2/login/login.ftl:165]:
```suggestion
                                                <title id="title">Stack Overflow</title>
```

File: themes/src/main/resources/theme/keycloak.v2/login/login.ftl [hunk 5]
```diff
@@ -119,63 +124,75 @@
...
                                             </svg>
                                             <#break>
                                         <#case "paypal">
                                             <svg viewBox="0 0 384 512" aria-hidden="true">
+                                                <title id="title">Paypal</title>
```
Comment (ahus1) [inline themes/src/main/resources/theme/keycloak.v2/login/login.ftl:195]:
```suggestion
                                                <title id="title">PayPal</title>
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Several brand names should need fixing of the capitalization. The test failure might be related, not sure.

**Review by ahus1 — APPROVED** [review]
Thank you, the brand names now look good!

**Pull request merged**